manifests/master/tcpwrappers.pp

```diff
@@ -5,17 +5,21 @@
 # Automatically included by profile_xcat::master
 class profile_xcat::master::tcpwrappers {
```

Comment:
Again, is this even really necessary. Most deployments that will use will not support tcpwrappers.

Reply:
Agree. Will open an issue to remove tcpwrappers since it's deprecated.

Reply:
Opened #2